components/infieldbutton/themes/express.css

```diff
@@ -0,0 +1,30 @@
+/*!
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+@container (--system: express) {
+  .spectrum-InfieldButton {
+    --spectrum-infield-button-border-width: 0;
+    --spectrum-infield-button-border-color: transparent;
+
+    --spectrum-infield-button-border-radius: var(--spectrum-corner-radius-75);
+    --spectrum-infield-button-border-radius-reset: var(--spectrum-in-field-button-fill-stacked-inner-border-rounding);
+
+    --spectrum-infield-button-stacked-top-border-radius-start-start: var(--spectrum-corner-radius-75);
+    --spectrum-infield-button-stacked-bottom-border-radius-end-start: var(--spectrum-corner-radius-75);
+
+    --spectrum-infield-button-background-color: var(--spectrum-gray-200);
+    --spectrum-infield-button-background-color-hover: var(--spectrum-gray-300);
+    --spectrum-infield-button-background-color-down: var(--spectrum-gray-400);
+    --spectrum-infield-button-background-color-key-focus: var(--spectrum-gray-300);
+
```

Comment:
I am seeing a hover background color change on the disabled variant when looking at the express theme, should we prevent that in this file?